ansible/roles/test/tasks/telemetry.yml

```diff
@@ -0,0 +1,176 @@
+- debug: msg="String telemetry dial-in and dial-out tests..."
+
+
+- set_fact:
+    path_tmp_telemetry: /tmp/tmp_telemetry
+    path_ptf_run_dir: /tmp/test_telemetry
+    gitUrl_telemetry: https://github.com/Azure/sonic-telemetry.git
+    version_telemetry: aaa9188fda4b6f36dd5da00cdb240933fedae5ce
+    gitUrl_mgmt_framework: https://github.com/Azure/sonic-mgmt-framework.git
+    version_mgmt_framework: a4f1650d505c14d94872bd216f218e7c752de05c
+    go_fileName: go1.14.linux-amd64.tar.gz
+
+- name: Create tmp folder for telemetry
+  delegate_to: localhost
+  file: path="{{ path_tmp_telemetry }}" state=directory
+
+
+ #****************************************#
+ #           Prepareing                   #
+ #****************************************#
+- debug: msg="Prepareing the test code folder..."
+- block:
+  - name: pip pyangbind
+    delegate_to: localhost
+    become: true
+    pip:
+      name: pyangbind==0.6.0
+  - name: install pip3
+    delegate_to: localhost
+    become: true
+    apt:
+      name: python3-pip
+  - name: pip jinja2
+    delegate_to: localhost
+    become: true
+    pip:
+      name: jinja2>=2.10
+      executable: pip3

```

Comment:
i do not know why this is needed? i think they are in the sonic-mgmt docker.

Reply:
sonic-mgmt-framework/tools/ui_gen/src/uigen.py needs jinja2 from python3